ui/src/app/components/table-column-changes-preview/table-column-changes-preview.component.scss

```diff
@@ -0,0 +1,21 @@
+.full-width {
```

Comment:
since all the tables in the Ui should follow similar CSS please move these local CSS to styles.css as much as possible with meaningful class names

Reply:
updated